projectDocs/dev/createDevEnvironment.md

```diff
@@ -7,6 +7,10 @@ If you plan on contributing to NVDA, you should [fork and clone](https://docs.gi
 
 Use the `--recursive` option when performing `git clone` to fetch the required git submodules we use.
 
+```
+git clone --recursive https://github.com/nvaccess/nvda.git

```

Comment:
this isn't quite right - contributors generally need to fork NVDA first, and clone that report

```suggestion
git clone --recursive https://github.com/username/nvdaRepo.git
```

Reply:
See steps above on line 6